audit/format.go

```diff
@@ -212,6 +218,11 @@ func (f *AuditFormatter) FormatResponse(ctx context.Context, w io.Writer, config
 			NumUses:                   resp.Auth.NumUses,
 			EntityID:                  resp.Auth.EntityID,
 			TokenType:                 resp.Auth.TokenType.String(),
+			TokenTTL:                  int64(resp.Auth.TTL.Seconds()),
+		}
+		var zt time.Time
+		if resp.Auth.IssueTime != zt {
+			respAuth.TokenIssueTime = resp.Auth.IssueTime.Format(time.RFC3339)
```

Comment:
I'm not sure why this is useful to include? Does it differ significantly from the time on the response itself?

Reply:
Hm, could be useful for renew calls, I guess?

Reply:
On a login I guess it would be a similar time.  I was just trying to add it consistently.